docs/units/equivalencies.rst

```diff
@@ -479,6 +479,13 @@ for them::
   >>> u.l.to(u.kg, 1, equivalencies=liters_water)
   1.0
 
+.. note::
+
+    To avoid infinite recursion of using ``units`` within ``units``
```

Comment:
I think I'd just add a sentence to the end of l.468: "The functions must take and return non-Quantity, to avoid infinite recursion of using ``units`` within ``units``."

Reply:
What about now?